src/mono/wasm/debugger/DebuggerTestSuite/DebuggerTestBase.cs

```diff
@@ -81,6 +81,10 @@ public DebuggerTestBase(string driver = "debugger-driver.html")
             scripts = SubscribeToScripts(insp);
 
             startTask = TestHarnessProxy.Start(FindChromePath(), DebuggerTestAppPath, driver);
+
+            // the debugger is working in locale of the debugged application. For example Datetime.ToString()
+            // we want the test to mach it. We are also starting chrome with --lang=en-US
+            System.Globalization.CultureInfo.CurrentCulture = new System.Globalization.CultureInfo("en-US");
```

Comment:
Should this be done in the static ctor?

Reply:
This is unit test, we don't have to optimize here, I think.

Reply:
Not for optimization. I was just thinking if anything else might be affected because we set it a bit late

Reply:
I see, sorry. I tried to do it in the static constructor already but it didn't work.
xUnit is messing with threads so it needs to be done on same thread as the asserts, therefore later in the setup (or twice).

Reply:
@lewing is there another way to do it?

We can move this to the beginning of the method, I think.

Reply:
moved